webapp/src/components/App/Tasks/index.tsx

```diff
@@ -456,6 +456,7 @@ function JobsListing() {
         date: task.completion_date_utc || task.creation_date_utc,
         type: task.type || "UNKNOWN",
         status: task.status === TaskStatus.Running ? "running" : "",
+        userName: (task.owner && usersByID[task.owner]?.name) || "",
```

Comment:
Is userName display needed only for tasks ? I think you should also add it for jobs (but I am not aware of the specs)